# Bioinformatics Notes

gProfiler:https://biit.cs.ut.ee/gprofiler/gost    gene set enrichment

ChromHMM: http://compbio.mit.edu/ChromHMM/

GREAT predict functions of cis-regulatory regions: http://bejerano.stanford.edu/great/public/html/index.php

FOCS enhancer and promoter: http://acgt.cs.tau.ac.il/focs/

3DIV 3D genome databse: http://kobic.kr/3div/

NGS Workshop 2019: https://angus.readthedocs.io/en/2019/toc.html

HGNC: protein-coding gene

Single-cell online book: https://osca.bioconductor.org/

RSEM (TPM): https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-323

RNA velocity (kallisto): https://bustools.github.io/BUS_notebooks_R/velocity.html

Gene-Drug: http://www.dgidb.org/search_interactions

Visualize Network: https://www.omicsnet.ca/

MethyCNV: http://bioconductor.org/packages/release/bioc/vignettes/conumee/inst/doc/conumee.html

SOM: #https://clarkdatalabs.github.io/soms/SOM_NBA

Splicing: rMATs

scATAC-seq: https://www.bioconductor.org/packages/devel/bioc/vignettes/cicero/inst/doc/website.html#references

Breast Cancer Survival: http://kmplot.com/analysis/

Cell-Cell: https://www.bioconductor.org/packages/devel/bioc/vignettes/scTensor/inst/doc/scTensor.html

deepTools: https://deeptools.readthedocs.io/en/latest/content/tools/plotHeatmap.html

g:Profiler: https://biit.cs.ut.ee/gprofiler/gost

Homer Motif: http://homer.ucsd.edu/homer/download.html

WGCNA: an R package for weighted correlation network analysis: https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/

CisTrome: http://cistrome.org/#

WashU browser: http://epigenomegateway.wustl.edu/legacy/

XenaBrowser: https://xenabrowser.net/datapages/

TCGA ATAC: https://gdc.cancer.gov/about-data/publications/ATACseq-AWG

Enrichr: http://amp.pharm.mssm.edu/Enrichr/

GeneMANIA: https://genemania.org/

PathwayCommons: https://www.pathwaycommons.org/

PathView: https://pathview.uncc.edu/

The Human Protein Atlas: https://www.proteinatlas.org/

R2 Cancer Database: https://hgserver1.amc.nl/cgi-bin/r2/main.cgi

Sequence Logo: https://weblogo.berkeley.edu/logo.cgi

Sequence Logo (compare): http://www.twosamplelogo.org/cgi-bin/tsl/tsl.cgi

VizioMetrics: http://viziometrics.org/search/

3D Hotspots: http://www.3dhotspots.org/#/home

OncoKB: http://oncokb.org/#/about

Personalized Cancer Therapy: https://pct.mdanderson.org/

FASMIC: http://ibl.mdanderson.org/fasmic/#!/

Allen Brain Atlas: http://www.brain-map.org/

Ecological Data Wiki: https://ecologicaldata.org/home

Marcher: https://cran.r-project.org/web/packages/marcher/vignettes/marcher.html

Ocean Biogeographic Information System: http://www.iobis.org/ , http://www.iobis.org/manual/visualization/

UCSC ENCODE composite: http://hgdownload.cse.ucsc.edu/goldenpath/hg19/encodeDCC/wgEncodeUwDgf/

Cell line microarray: http://cellexpress.cgm.ntu.edu.tw/

IntoGen: https://www.intogen.org

GEPIA: http://gepia.cancer-pku.cn/index.html

CoExpress: http://coxpresdb.jp, http://www.genefriends.org/RNAseq/

TF-JASPAR: http://jaspar.genereg.net

MicroRNA: http://mirtarbase.mbc.nctu.edu.tw/php/index.php

LINCS: http://www.lincsproject.org/LINCS/

3DSNP: http://cbportal.org/3dsnp/

TF CHIP-seq database GTRD: http://gtrd.biouml.org

TF Target Gene database TRRUST: http://www.grnpedia.org/trrust/

Tissue Specific Expression Analysis (TSEA): http://genetics.wustl.edu/jdlab/tsea/

Cell Type CTEN: http://www.influenza-x.org/~jshoemaker/cten/

Broad Institute Cancer Cell Line: https://portals.broadinstitute.org/ccle

Cancer Genetiv Web: http://www.cancer-genetics.org/

Single Cell database: http://single-cell.clst.riken.jp/, https://portals.broadinstitute.org/single_cell'

R mclust: https://cran.r-project.org/web/packages/mclust/index.html

R princurve: https://cran.r-project.org/web/packages/princurve/index.html

R mixtools: https://cran.r-project.org/web/packages/mixtools/vignettes/mixtools.pdf

Gene Fusion FACTERA: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4296148/

TCGA analysis cbioportal: http://www.cbioportal.org/

http://gdac.broadinstitute.org/#, http://bioinformatics.mdanderson.org/TCGA/NGCHMPortal/

https://bioconductor.org/packages/release/bioc/vignettes/vidger/inst/doc/vidger.html

http://www.cistrome.org/

https://deepcognition.ai/products/desktop/

Name: https://www.nameacronym.net/, https://acronymify.com/

https://blog.csdn.net/kmd8d5r/article/details/80879161


ccRemover: https://cran.r-project.org/web/packages/ccRemover/vignettes/ccRemover_tutorial.html

RNASeqPower: https://bioconductor.org/packages/release/bioc/manuals/RNASeqPower/man/RNASeqPower.pdf

Motif compare: http://rsat.sb-roscoff.fr/

CNN Motif: https://github.com/davek44/Basset

Github: https://blog.csdn.net/emaste_r/article/details/19560321

Github RM : https://wodewone.github.io/2016/11/27/git%E6%89%B9%E9%87%8F%E5%88%A0%E9%99%A4%E6%96%87%E4%BB%B6%E5%A4%B9%E5%92%8C%E6%96%87%E4%BB%B6/

KEGG visualization: Pathview Web: user friendly pathway visualization and data integration

Homogeneity of variance test: https://www.jianshu.com/p/dc8896fcd505
